### ISSUE TITLE
VTIMEZONE block in output stream

### ISSUE THREAD

**Issue by Omkommersind** [jasvrcek/ICS#27]
Hello, is there a way not to put this such blocks:

> BEGIN:VTIMEZONE\r\n
> TZID:Europe/Moscow\r\n
> BEGIN:STANDARD\r\n
> DTSTART:19691231T210000\r\n
> TZOFFSETTO:+0300\r\n
> TZOFFSETFROM:+0300\r\n
> END:STANDARD\r\n
> END:VTIMEZONE\r\n

in output stream?

And what purpose of adding this information?

**Comment by BernhardWebstudio** [comment]
AS far as I understand, it is neccessary to give the calendar an indication how to interpret the times and dates in the file. To set the timezone, you can call `setTimezone(new DateTimeZone('Your/Timezone'))` on your Calendar object. I think the default one is [America/New_York](https://github.com/jasvrcek/ICS/blob/master/src/Model/Calendar.php), which I find rather unsuitable, as it is neither the "±0"-timezone, nor is it fetched from php.ini via `date_default_timezone_get()`. @jasvrcek is my observation correct? Could we change this in one or another way?

**Comment by jasvrcek** [comment]
@BernhardWebstudio that is correct, you need that block to tell the calendar how to interpret dates. Changing the default timezone to UTC would be fine with me. In my typical use case the timezone needs to match that of a given application user, rather than whatever the server's timezone happens to be, so I would vote against having it default to the default set in php.ini.

**Comment by mrcknblch** [comment]
Regarding the transition part when adding the VTIMEZONE - would't it make sense to use the current calendar event dates as scope for getting the Transitions instead of 1970's start and end? 

Stumbled over it when emails with Outlook events broke on the DST issue and where dated forward about one hour - though ics-files build the same way where imported correctly.
